src/index.html

```diff
@@ -7,27 +7,29 @@
         <!-- FONT LINK -->
         <link rel="preconnect" href="https://fonts.googleapis.com" />
         <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+        <link rel="preconnect" href="https://fonts.googleapis.com" />
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
         <link
-            href="https://fonts.googleapis.com/css2?family=Black+Ops+One&family=Merriweather:wght@400;900&display=swap"
+            href="https://fonts.googleapis.com/css2?family=Luckiest+Guy&display=swap"
             rel="stylesheet"
         />
         <title>Castle Game</title>
     </head>
     <body>
         <div id="container">
             <div id="start-page">
-                <h1 id="game-title" class="title">
+                <!-- <h1 id="game-title" class="title">
                     Math Cast<span id="le">le</span>
-                </h1>
-                <div id="instructions">
+                </h1> -->
+                <!-- <div id="instructions">
                     <h2 class="instructions-header">Instructions:</h2>
                     <p class="instructions-paragraph">
                         Lorem ipsum dolor sit amet consectetur adipisicing elit.
                         Magnam nulla explicabo magni sequi ipsam ad atque? Amet
                         vitae minus non optio consequatur iste. Molestiae dolore
                         quasi saepe, officiis necessitatibus modi!
                     </p>
-                </div>
+                </div> -->
```

Comment:
Need to remove all the comments

Reply:
We need a name and instructions of the game so these comments could stay here temporary